README.md

```diff
@@ -29,15 +29,32 @@ avoids breaking changes other than at major (x.0.0) releases. However,
 occasionally Azure API fixes require breaking updates within an individual
 package; these exceptions are noted in release changelogs.
```

Comment:
maybe we can merge this sentence with the next one you added ?

Reply:
I think they're about different things. The previous sentence is talking about breaking changes *within* a major, the next sentence is talking about us releasing new majors frequently.

Do you perhaps mean combine the paragraphs? I prefer to break up paragraphs pretty frequently cause I believe people tend to skim the end of long paragraphs, so I'd rather keep them separate.

Reply:
yes but the thing is we are actually not doing that right, we have monthly releases on which we bump the major version

Reply:
true, but the first sentence is to notify people in advance of the reality. perhaps we no longer do this cause we can defer to end of the month and latest? then I can take the sentence out.